### DISCUSSION TITLE
wake on lan not working

### DISCUSSION THREAD

**Discussion by excelsior091224** [AlexGustafsson/homebridge-wol#271] [General]
### Issue

I'm using homebridge installed on LXC on Proxmox, but Wake on Lan has stopped working for at least the past week. The settings of the PC I want to wake up with Wake on Lan haven't changed at all since it worked, so I think it's a problem with the plugin.

### Environment

```Markdown
- os and os version:Debian GNU/Linux Bookworm (12)
- node version:v22.12.0
- homebridge version:v2.0.0-beta.23
- homebridge-wol version:v7.0.0-beta.0
```

### Logs

```Markdown
When I follow these steps:
1. Make sure `logLevel` is set to `Debug` on all `NetworkDevice` accessories
2. Run `homebridge`
3. Let it run for a couple of minutes
4. Try to turn a device on or off using the plugin
5. Wait one minute
6. Repeat step 4

I get the following log:

[2024/12/26 21:22:52] [ただしのデスクトップPC] [NetworkDevice] Awake cycle started
[2024/12/26 21:22:52] [ただしのデスクトップPC] [NetworkDevice] Attempting to wake up device
[2024/12/26 21:22:52] [ただしのデスクトップPC] [NetworkDevice] Device went from status "Offline" to "WakingUp"
[2024/12/26 21:22:53] [ただしのデスクトップPC] [NetworkDevice] Waiting for wake grace time (45000ms)
[12/26/2024, 9:22:54 PM] [Google Smart Home] Sending State Report
[12/26/2024, 9:22:54 PM] [Google Smart Home] {
  "type": "report-state",
  "body": {
    "f206e3f34942229aeddba9f2d854bee9892104bb63237daf095f6670cb719fdc": {
      "on": true,
      "online": true
    }
  }
}
[2024/12/26 21:23:39] [ただしのデスクトップPC] [NetworkDevice] Device went from status "WakingUp" to "Offline"
[12/26/2024, 9:23:40 PM] [Google Smart Home] Sending State Report
[12/26/2024, 9:23:40 PM] [Google Smart Home] {
  "type": "report-state",
  "body": {
    "f206e3f34942229aeddba9f2d854bee9892104bb63237daf095f6670cb719fdc": {
      "on": false,
      "online": true
    }
  }
}

```
```


### Notes

_No response_

**Comment by AlexGustafsson** [comment]
Did everything work before? Did you use the beta versions of home bridge and home bridge-wol at that point? The beta version of the plugin was published in October. There's been no code changes since the previous version, which was published in January - soon a year ago. So the plugin hasn't changed.

**Comment by excelsior091224** [comment]
> Did everything work before? Did you use the beta versions of home bridge and home bridge-wol at that point? The beta version of the plugin was published in October. There's been no code changes since the previous version, which was published in January - soon a year ago. So the plugin hasn't changed.

Yes, I updated to 7.0 beta today, but it still doesn't work. I was using the latest stable version until then. Also, after creating the issue, I moved from LXC on Proxmox to RaspberryPi4, but it still doesn't seem to work there. My current environment is as follows.

OS:Debian GNU/Linux Bookworm (12)(RaspberryPiOS)
node.js:v22.12.0
homebridge:v1.8.5
homebridge-wol:v7.0.0-beta.0

**Comment by AlexGustafsson** [comment]
I'm sorry, I can't see why there would be any issues with the plugin. It hasn't changed for a year and you say it worked for you in the past. I'll convert your issue to a discussion and keep it open to see if anyone else might have experienced the same issue you have.

**Comment by excelsior091224** [comment]
I put the wakeonlan command into the raspberry pi with `sudo apt -y install wakeonlan` and sent a magic packet to the target PC (Windows 11), but it didn't wake up. It seems to be a problem with the Windows machine, not the plugin. I haven't changed the settings in the Bios or the Device Manager for the network adapter, so it might be that a recent Windows update has made it impossible to wake up.

**Comment by excelsior091224** [comment]
I turned off one of the proxmox clusters from homebridge and sent a magic packet, and it worked fine. It's definitely a problem on the Windows side. Damn Microsoft, you're doing something unnecessary.
